src/components/button/button.style.tsx

```diff
@@ -45,7 +45,7 @@ export const getStyle = (theme: ThemeType, props: ButtonProps) => {
     ...createBorderRadiusStyles(props, theme.borderRadius),
     backgroundColor: getThemeColor(
       theme.colors,
-      props.bg || `${props.colorScheme}.500`
+      `${props.bg ?? props.colorScheme}.500`
```

Comment:
I think you don't want to do that. bg and colorScheme are two different things. If I put `gray.600` as a background I don't want the value to be `gray.600.500`

Reply:
So what you're saying is we should expect Button bg prop to be called with a shade and colorScheme a color ? Like `<Button bg="red.500" colorScheme="blue">`  ?

Reply:
I mean that final developer shouldn't be able to put `bg` and `colorScheme` (in fact they can, but some precedence rule will override the `colorScheme` with the `bg`. `bg` and `colorScheme` are two different things. `bg` is used to put precise values, `colorScheme` is used to provide a color and then the component will adapt itself with it using the [0, ..., 900] palette.